tests/integration/targets/django_manage/tasks/main.yaml

```diff
@@ -9,17 +9,10 @@
     suffix: .django_manage
   register: tmp_django_root
 
-- name: Install virtualenv on CentOS 8
+- name: Install virtualenv
   package:
-    name: virtualenv
+    name: "{{ os_package_name.virtualenv }}"
```

Comment:
What happens if this is undefined? For example for Debian or RHEL? Shouldn't there be a `when` condition for these cases?

Reply:
The `setup_os_pkg_name` should have at least a default value. Right now it is just for virtualenv, but other packages might benefit from it.

Reply:
And this is not user-facing code, so the developer will get an error and will need to add the correct values.